### PR TITLE
Added en translation for Ellipsis

### PR DESCRIPTION
**Pull request by cemremengu**

### CODE REVIEW

File: src/components/Ellipsis/index.en-US.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Ellipsis 
+subtitle: Ellipsis text
```
Comment (ddcat1115) [inline src/components/Ellipsis/index.en-US.md]: Redundant line

File: src/components/Ellipsis/index.en-US.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+cols: 1
+order: 10
+---
+
+When the text is too long, the ellipsis automatically shortens it according to its length and the maximum number of lines.
```
Comment (ddcat1115) [inline src/components/Ellipsis/index.en-US.md]: ellipsis should be capitalized.

`and` -> `or`

**Comment by ddcat1115** [comment]
Demo translation missing: https://github.com/ant-design/ant-design-pro/tree/master/src/components/Ellipsis/demo

**Comment by cemremengu** [comment]
@ddcat1115 Added demo translation. Can you verify that the format is correct so I can continue similarly with the others?

**Comment by ddcat1115** [comment]
The Format is right~

**Pull request merged**